dhcpv6/options.go

```diff
@@ -106,6 +106,8 @@ func ParseOption(dataStart []byte) (Option, error) {
 		opt, err = ParseOptClientArchType(optData)
 	case OptionNII:
 		opt, err = ParseOptNetworkInterfaceId(optData)
+	case OptionVendorOpts:
```

Comment:
I would sort this by option number instead of adding to the end. How about poting this after OptionVendorClass.